learn/functions/return_function.md

```diff
@@ -0,0 +1,188 @@
+
+Returning a function from a function means that a function is created and returned as output of another function. This is possible in Python because functions are first-class objects, which means they can be passed around just like any other object such as integers, strings, or lists.
+
+## Return function in Python
```

Comment:
Add a text between H2 and H3 or we can remove this H2 and make further H3 => H2.